packages/dds/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ soss_install_middleware_plugin(
 )
 
 install(
-    FILES ${PROJECT_SOURCE_DIR}/sample/tcp/config.xml
+    FILES ${PROJECT_SOURCE_DIR}/../../sample/tcp/config.xml
```

Comment:
It should be `../../examples/` or something like that?

Reply:
Right!

Reply:
```suggestion
    FILES ${PROJECT_SOURCE_DIR}/../../examples/tcp/config.xml
```